packages/orchestration/test/exos/local-chain-account-kit.test.ts

```diff
@@ -0,0 +1,133 @@
+  const localchain = makeLocalChain({
+    bankManager,
+    system: fakeBridgeKit.bridgeHandler,
+  });
+  const timer = buildManualTimer(t.log);
+  const marshaller = makeFakeBoard().getReadonlyMarshaller();
+
+  // ChainInfo mocked via `prepareMockChainInfo` until #8879
+  const agoricChainInfo = prepareMockChainInfo(makeBaggageAndZone().zone);
+
+  // exo setup
+  const { baggage } = makeBaggageAndZone();
+  const { makeRecorderKit } = prepareRecorderKitMakers(baggage, marshaller);
+  // mocked ZCF. use `stake-bld.contract.test.ts` to test LCA with offers
+  const mockZcf = Far('MockZCF', {});
+  const makeLocalChainAccountKit = prepareLocalChainAccountKit(
+    baggage,
+    makeRecorderKit,
+    // @ts-expect-error mocked zcf
+    mockZcf,
+  );
+
+  const lca = await E(localchain).makeAccount();
+  const address = await E(lca).getAddress();
+
+  const { holder: account } = makeLocalChainAccountKit({
+    account: lca,
+    address,
+    storageNode: makeMockChainStorageRoot().makeChildNode('lcaKit'),
+    timer,
+    timerBrand: timer.getTimerBrand(),
+    agoricChainInfo,
+  });
+
+  t.truthy(account, 'account is returned');
+  t.regex(await E(account).getAddress(), /agoric1/);
+
+  const oneHundredStakeAmt = stake.make(1_000_000_000n);
+  const oneHundredStakePmt = issuerKit.mint.mintPayment(oneHundredStakeAmt);
+  const oneStakeAmt = stake.make(1_000_000n);
+
+  // deposit 100 bld to account
+  const depositResp = await E(account).deposit(oneHundredStakePmt);
+  t.true(AmountMath.isEqual(depositResp, oneHundredStakeAmt), 'deposit');
+
+  const destination: ChainAddress = {
+    chainId: 'cosmoslocal',
+    address: 'cosmos1pleab',
+    addressEncoding: 'bech32',
+  };
+  // transfer 10 bld to cosmos using ERTP amount
+  await t.throwsAsync(() => E(account).transfer(oneStakeAmt, destination), {
+    message: 'ERTP Amounts not yet supported', // TODO #9211, support ERTP amounts
+  });
+  const transferResp = await E(account).transfer(
+    { denom: 'ubld', value: 10n },
+    destination,
+  );
+  t.is(transferResp, undefined, 'Successful transfer returns Promise<void>.');
+
+  await t.throwsAsync(
+    () => E(account).transfer({ denom: 'ubld', value: 504n }, destination),
+    {
+      message: 'simulated MsgTransfer packet timeout',
+    },
+  );
+
+  const unknownDestination: ChainAddress = {
+    chainId: 'unknown',
+    address: 'uknown1pleab',
+    addressEncoding: 'bech32',
+  };
+
+  await t.throwsAsync(
+    () => E(account).transfer({ denom: 'ubld', value: 1n }, unknownDestination),
+    {
+      message: 'key "unknown" not found in collection "ibcConnections"',
+    },
+    'cannot create transfer msg with unknown chainId',
+  );
+
+  await t.notThrowsAsync(
+    () =>
+      E(account).transfer({ denom: 'ubld', value: 10n }, destination, {
+        memo: 'hello',
+      }),
+    'can create transfer msg with memo',
+  );
+  // TODO, can we intercept the bridge message to see that it has a memo?
```

Comment:
cc @turadg

Reply:
yeah. I could add that to https://github.com/Agoric/agoric-sdk/pull/9396 but I wouldn't have a case to drive it. I'd prefer to sequence that first and add the functionality here. Consider rebasing this branch on top of `9207-bridge`